client/src/includes/initSkipLink.ts

```diff
@@ -1,17 +1,25 @@
 const initSkipLink = () => {
-  // Inspired by https://github.com/selfthinker/dokuwiki_template_writr/blob/master/js/skip-link-focus-fix.js
+  /**
+   * Appears at the top left corner of the admin page with the tab button is clicked.
+   * Used to provide an accessible skip button for keyboard control so that users can easily navigate to the main content without having to navigate a long list of navigation links.
+   *
+   * Inspired by https://github.com/selfthinker/dokuwiki_template_writr/blob/master/js/skip-link-focus-fix.js
+   *
+   */
 
```

Comment:
This JSDOC needs to be above the function declaration to be used correctly by code editors.

Reply:
Noted